docs/multicluster/antctl.md

```diff
@@ -26,5 +30,59 @@ antctl mc get clusterset [NAME] [-n NAMESPACE] [-o json|yaml] [-A]
 antctl mc get resourceimport [NAME] [-n NAMESPACE] [-o json|yaml] [-A]
 antctl mc get resourceexport [NAME] [-n NAMESPACE] [-clusterid CLUSTERID] [-o json|yaml] [-A]
 ```
-
 To see the usage examples of these commands, you may also run `antctl mc get [subcommand] --help`.
+
+## antctl mc create
+
+`antctl mc create` command can create access-token and other resources like ClusterSet, ClusterClaims for 
+Antrea Multi-cluster setup
+
++ `antctl mc create accesstoken` command can create accesstoken for member clusters. 
++ `antctl mc create clusterclaims` command can create two ClusterClaims in a leader or member cluster. One for the leader or member cluster, and another for the ClusterSet.
++ `antctl mc create clusterset` command can create a ClusterSet in a leader or member cluster.
+
+```bash
+$ anctcl mc create accesstoken [NAME] [-n NAMESPACE] [--serviceaccount SERVICE_ACCOUNT] [--role-binding ROLE_BINDING]
+$ antctl mc create clusterclaims [-n NAMESPACE] [--clusterset-id CLUSTERSET_ID] [--cluster-id CLUSTER_ID]
+$ antctl mc create clusterset [NAME] [-n NAMESPACE] [--leader-server LEADER_SERVER] [--service-account SERVICE_ACCOUNT] [--secret SECRET] [--leader-cluster LEADER_CLUSTER_ID]
+```
+To see the usage examples of these commands, you may also run `antctl mc create [subcommand] --help`.
+
+## antctl mc add
+
+`antctl mc add` command can add a new member cluster to a ClusterSet.
+
+```bash
+$ antctl mc add member-cluster [CLUSTER_ID] [-n NAMESPACE] [--clusterset CLUSTERSET] [--service-account SERVICE_ACCOUNT]
```

Comment:
This command seems different from other subcommands, objects in other subcommands don't connect words with "-". AFAIK, all kubectl and antctl don't use "-" in objects, maybe it should keep align to make the command easier to remember.

Reply:
Done. Removed the "-" in the "leader-cluster" and the "member-cluster" in `deploy/leader_cluster.go`, `deploy/member_cluster.go`, the `add/member_cluster.go` and the `delete/member_cluster.go`

Reply:
the example is not updated

Reply:
Done